### PR TITLE
Update config for tx-pool-limit-by-account-percentage on pub nets, consider moving to both sides 

### PR DESCRIPTION
**Pull request by non-fungible-nelson**
This updates the tx-pool-limit to the correct name for the parameter and provides a word of caution. Suggest that we copy this over to the private network side and call it out as well until we have a better parameterization default. We may be changing this shortly on Mainnet. I will ask @fab-10 to tag the docs team when he finishes his new transaction pool work. 

Signed-off-by: Matt Nelson <85905982+non-fungible-nelson@users.noreply.github.com>

### CODE REVIEW

**Review by alexandratran — APPROVED** [review]
Minor edits and added a note on the private networks start node page. LGTM.

**Review by macfarla — APPROVED** [review]

File: docs/public-networks/reference/cli/options.md [hunk 1]
```diff
@@ -3302,34 +3302,41 @@ Use the [`miner_changeTargetGasLimit`](../api/index.md#miner_changetargetgaslimi
...
 
     ```bash
-    tx-pool-future-max-by-account="100"
+    tx-pool-limit-by-account-percentage=0.4
     ```
 
-The maximum number of future transactions kept in the transaction pool, per account.
-The default is 64.
+The maximum percentage of future transactions kept in the transaction pool, per account.
+The default is .1 or 10% of transactions from a single account to be kept in the pool.
```
Comment (fab-10) [inline docs/public-networks/reference/cli/options.md:3332]: The current default is .001 or 0.1%, that for the default size means max 5 txs per sender

**Review by fab-10 — CHANGES_REQUESTED** [review]
Made 2 corrections

File: docs/public-networks/reference/cli/options.md [hunk 2]
```diff
@@ -3302,34 +3302,41 @@ Use the [`miner_changeTargetGasLimit`](../api/index.md#miner_changetargetgaslimi
...
 `target-gas-limit` value.
 
-### `tx-pool-future-max-by-account`
+### `tx-pool-limit-by-account-percentage`
 
 === "Syntax"
 
     ```bash
-    --tx-pool-future-max-by-account=<INTEGER>
+    --tx-pool-limit-by-account-percentage=<LONG>
```
[the remark this answers is not in the document]
Comment (fab-10) [reply docs/public-networks/reference/cli/options.md:3310]: we can also says that accepted values are in the range (0-1]

**Comment by non-fungible-nelson** [comment]
@alexandratran any status?

**Comment by alexandratran** [comment]
@non-fungible-nelson Sorry for the delay. Can you make the changes that @fab-10 requested? Then he needs to re-review. Otherwise I can make the changes probably tomorrow.

**Comment by non-fungible-nelson** [comment]
I thought i accepted his changes? @fab-10 can you re-review?

**Review by non-fungible-nelson — COMMENTED** [review]
accepting changes

**Comment by alexandratran** [comment]
@non-fungible-nelson no, he commented his change suggestions but didn't implement them. It looks like you resolved his comment without implementing.

File: docs/public-networks/reference/cli/options.md [hunk 3]
```diff
@@ -3331,34 +3331,41 @@ Use the [`miner_changeTargetGasLimit`](../api/index.md#miner_changetargetgaslimi
...
```
Comment (fab-10) [inline docs/public-networks/reference/cli/options.md:3339]:
```suggestion
    --tx-pool-limit-by-account-percentage=<DOUBLE>
```

File: docs/public-networks/reference/cli/options.md [hunk 4]
```diff
@@ -3331,34 +3331,41 @@ Use the [`miner_changeTargetGasLimit`](../api/index.md#miner_changetargetgaslimi
...
 
     ```bash
-    tx-pool-future-max-by-account="100"
+    tx-pool-limit-by-account-percentage=0.4
     ```
 
-The maximum number of future transactions kept in the transaction pool, per account.
-The default is 64.
+The maximum percentage of future transactions kept in the transaction pool, per account.
+The default is .1 or 10% of transactions from a single account to be kept in the pool.
```
Comment (fab-10) [inline docs/public-networks/reference/cli/options.md:3361]:
```suggestion
The maximum percentage of future transactions kept in the transaction pool, per account.
Accepted values are in the range (0-1]. The default is .001 or 0.1% of transactions from a single account to be kept in the pool.
```

**Comment by fab-10** [comment]
@non-fungible-nelson please check my suggested change and incorporate them

**Review by fab-10 — APPROVED** [review]

**Pull request merged**